feeding/src/FeedingDemo.cpp

```diff
@@ -182,8 +183,13 @@ FeedingDemo::FeedingDemo(
   mTiltOffset = Eigen::Vector3d(
       tiltOffsetVector[0], tiltOffsetVector[1], tiltOffsetVector[2]);
 
-  mVelocityLimits
+  std::vector<double> velocityLimits
       = getRosParam<std::vector<double>>("/study/velocityLimits", *mNodeHandle);
+  while (velocityLimits.size() < 6)
```

Comment:
Not huge, but imo it'd be clearer (and more concise) to have something like
```cpp
velocityLimits = ...
mVelocityLimits = 0.2 * Eigen::Vector6d::Ones();
for (std::size_t i = 0; i < velocityLimits.size(); ++i)
  mVelocityLimits[i] = velocityLimits[i];
```
It could also help to have a comment to explain why `/study/velocityLimits` might be shorter than 6?